.github/workflows/ci.yml

```
@@ -2,8 +2,16 @@ name: CI
 on:
   push:
     branches: ["*"]
+    paths:
+      - 'src/**'
+      - 'resources/**'
+      - 'Cargo.*'
```

```suggestion
    paths-ignore:
      - "docs/**"
      - "**.md"
```

It may be safer that way.

---

Yeah, I was debating between the two, but you're probably right. But there are a lot of other paths to ignore: assets, config files, etc. But this should prevent the majority of unnecessary runs, so applying.